src/service/action-impl.js

```diff
@@ -121,6 +126,39 @@ export class ActionService {
   }
 
   /**
+   * Installs action handler for the specified element.
+   * @param {!Element} target
+   * @param {function(!ActionInvocation)} handler
+   */
+  installActionHandler(target, handler) {
+    const debugid = target.tagName + '#' + target.id;
+    assert(this.isAmpElement_(target), 'AMP element is expected: %s', debugid);
+
+    const currentQueue = target[ACTION_QUEUE_];
+    if (currentQueue) {
+      assert(Array.isArray(currentQueue), 'Expected queue to be an array: %s',
+          debugid);
+    }
+
+    // Override queue with the handler.
+    target[ACTION_QUEUE_] = {'push': handler};
```

Comment:
This may not be a problem yet, but we've created a possibly-sync/possibly-async handler here ([Zalgo](http://blog.izs.me/post/59142742143/designing-apis-for-asynchrony)). Maybe a `setTimeout` handler instead, so we're always async?